packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -126,6 +134,12 @@ const ModalManager = () => {
 
     return (
         <React.Fragment>
+            <WalletsConsentPopup
```

Comment:
This modal needs to be added as the last step to the wallets modal, please add a comment here to update this in stitching [user story](https://app.clickup.com/t/20696747/WALL-167)

Reply:
Done 😃 👍🏼